package.json

```diff
@@ -19,7 +19,7 @@
     "atom-space-pen-views": "^2.0.0",
     "underscore-plus": "^1.0.0",
     "xregexp": "^3.1.1",
-    "afterwriting": "^1.7.0",
+    "afterwriting": "^1.10.0",
```

Comment:
Apparently the `^` actually upgrades the minor version on its own when running `npm i`, but it never hurts to be clear.